### DISCUSSION TITLE
system/2020/10/09/upgrade-power8-bmc/

### DISCUSSION THREAD

**Discussion by giscus** [jiegec/blog-source#14] [General]
# system/2020/10/09/upgrade-power8-bmc/

杰哥的{运维，编程，调板子}小笔记

https://jia.je/system/2020/10/09/upgrade-power8-bmc/

**Comment by lbgws2** [comment]
‍大佬你好,有IBM Power S822LC(8335-GTB) 的问题想要咨询一下,能麻烦指导一下吗,我的微信lbgws2

**Reply by jiegec** [reply]
您在这里讲就可以，我对这台机器也没有很了解，现有信息都是在网络搜索得到的。

**Comment by lotest** [comment]
请教下！2017年的版本，登录到BMC GUI之后 configuration中找不到Power Redundancy Settings的选项，有没有办法可以设置电源？

**Reply by jiegec** [reply]
我也不太清楚，不妨问一下 IBM 的 Support？
